### DISCUSSION TITLE
Gulp Fail to load

### DISCUSSION THREAD

**Discussion by mirzajb** [Dotnet-Boxed/Templates#95] [Q&A]
i have tried to restore packages manually also unload and reload the project but nothing working always gives this .
![untitled](https://cloud.githubusercontent.com/assets/12029067/15091639/b3f2fcee-1403-11e6-95d6-3e2c46c0f5c3.png)

**Comment by RehanSaeed** [comment]
Go to the output window and select 'Task Runner Explorer' under 'show output from'. You will see what the error is. NPM or Microsoft's use of NPM is a bit flakey and sometimes fails to download packages correctly. You most likely need to clear your package cache and delete the node_modules folder. There is a note in the ReadMe at the top.

**Comment by RehanSaeed** [comment]
If you are still having problems, feel free to re-open this issue. Hopefully the ASP.Net Core RC2 tooling will fix these issues.
